src/operations/execute_operation.ts

```diff
@@ -51,36 +51,24 @@ export interface ExecutionResult {
  * @internal
  *
  * @remarks
- * This method reduces large amounts of duplication in the entire codebase by providing
- * a single point for determining whether callbacks or promises should be used. Additionally
- * it allows for a single point of entry to provide features such as implicit sessions, which
+ * Allows for a single point of entry to provide features such as implicit sessions, which
  * are required by the Driver Sessions specification in the event that a ClientSession is
- * not provided
+ * not provided.
  *
- * @param topology - The topology to execute this operation on
+ * The expectation is that this function:
+ * - Connects the MongoClient if it has not already been connected
+ *   - This amounts to creating and starting the Monitors, something that serverSelection could also do
```

Comment:
```suggestion
```

Reply:
Fair enough, wishful thinking 😅 I did want to capture that it is not a strict requirement to call "connect" but rather start the logic of connecting... anyway too much for implication something I can save for how CSOT comes together.